lib/Driver/Driver.cpp

```diff
@@ -1411,11 +1411,15 @@ void Driver::buildOutputInfo(const ToolChain &TC, const DerivedArgList &Args,
       OI.CompilerOutputType = file_types::TY_Remapping;
       OI.LinkAction = LinkKind::None;
       break;
+
+    case options::OPT_dump_ast:
+      OI.CompilerOutputType = file_types::TY_ASTDump;
+      break;
```

Comment:
Nitpick: can you put this next to the other "simple" cases, so it's clearer that they behave similarly?

Reply:
I think I put it there in part because it was next to (just before) the `Nothing` cases. Where would you rather I put it?

Reply:
I was thinking up after `OPT_emit_bc`, since that's where the rest of the `-o`-ish options are.